app/models/service_template_provision_task.rb

```diff
@@ -145,12 +145,12 @@ def deliver_to_automate(req_type = request_type, _zone = nil)
 
       zone ||= source.respond_to?(:my_zone) ? source.my_zone : MiqServer.my_zone
       MiqQueue.put(
-        :class_name  => 'MiqAeEngine',
-        :method_name => 'deliver',
-        :args        => [args],
-        :role        => 'automate',
-        :zone        => options.fetch(:miq_zone, zone),
-        :task_id     => "#{self.class.name.underscore}_#{id}"
+        :class_name     => 'MiqAeEngine',
+        :method_name    => 'deliver',
+        :args           => [args],
+        :role           => 'automate',
+        :zone           => options.fetch(:miq_zone, zone),
+        :tracking_label => "#{self.class.name.underscore}_#{id}"
```

Comment:
This seems duplicate to the "deliver_to_automate" above...can we extract and DRY up?